tests/unit/backend/fixtures/test_db.py

```diff
@@ -482,3 +482,29 @@ def _create_test_dataset_processing_status(self):
         )
         self.session.add(dataset_processing_status)
         self.session.commit()
+        dataset_processing_status = DbDatasetProcessingStatus(
```

Comment:
we should probably stop adding test data here and instead add it to the test file where it's used. This will speed up testing overall, and make it clear what test data is needed for a test. Can you move these to `test_curation_collections.py`. The `generate_dataset` function should help with this.

Reply:
If it's easy do it, otherwise we can do this in a larger refactor

Reply:
I'm kind of in favor of leaving in these additions because they complete/fill out other entities that are already in `test_db.py`.

But generally I would agree that we could benefit from a more modular way to initialize the db for each test suite. Probably need to review and decide on an ideal testing db "base" state.